packages/core/src/components/ozon-content/nodes/abbr.node.tsx

```diff
@@ -0,0 +1,19 @@
+import { h } from "@stencil/core";
+
+import { OzonContentNode } from "../ozon-content-node.interface";
+import { OzonContentNodeContext } from "../ozon-content-node-context.interface";
+
+export class OzonContentAbbrNode implements OzonContentNode {
+  name = "abbr";
+
+  render(node: Element, { mapNodeToJsx }: OzonContentNodeContext) {
+    const title = node.getAttribute("title");
+
+    return (
+      <span class="od-abbr">
+        <abbr>{mapNodeToJsx(node.childNodes)}</abbr>
+        {title ? ` (${title})` : null}
```

Comment:
Je kunt hier een AND operator gebruiken. Scheelt weer een conditie

```suggestion
        {title && ` (${title})`}
```

Reply:
Aangepast